docs/product/components/tables.html

```diff
@@ -631,6 +633,76 @@
+
+    {% capture html %}
+        <table class="s-table s-table__sortable" data-controller="s-table">
+            <thead>
+                <tr>
+                    <th data-target="s-table.column" data-action="click->s-table#sort">
+                        Season
+                        @Svg.ArrowUpSm.With("js-sorting-indicator js-sorting-indicator-asc d-none") <!-- CODE ONLY -->
+                        {% icon ArrowUpSm | js-sorting-indicator js-sorting-indicator-asc d-none %} <!-- PREVIEW ONLY -->
+                        @Svg.ArrowDownSm.With("js-sorting-indicator js-sorting-indicator-desc d-none") <!-- CODE ONLY -->
+                        {% icon ArrowDownSm | js-sorting-indicator js-sorting-indicator-desc d-none %} <!-- PREVIEW ONLY -->
+                        @Svg.ArrowUpDownSm.With("js-sorting-indicator js-sorting-indicator-none") <!-- CODE ONLY -->
+                        {% icon ArrowUpDownSm | js-sorting-indicator js-sorting-indicator-none %} <!-- PREVIEW ONLY -->
+                    </th>
+                    <th data-target="s-table.column" data-action="click->s-table#sort">
+                        Starts in month
+                        {% icon ArrowUpSm | js-sorting-indicator js-sorting-indicator-asc d-none %} <!-- PREVIEW ONLY -->
+                        {% icon ArrowDownSm | js-sorting-indicator js-sorting-indicator-desc d-none %} <!-- PREVIEW ONLY -->
+                        {% icon ArrowUpDownSm | js-sorting-indicator js-sorting-indicator-none %} <!-- PREVIEW ONLY -->
+                        … <!-- CODE ONLY -->
+                    </th>
+                    <th data-target="s-table.column" data-action="click->s-table#sort">
+                        Typical temperature in °C
+                        {% icon ArrowUpSm | js-sorting-indicator js-sorting-indicator-asc d-none %} <!-- PREVIEW ONLY -->
+                        {% icon ArrowDownSm | js-sorting-indicator js-sorting-indicator-desc d-none %} <!-- PREVIEW ONLY -->
+                        {% icon ArrowUpDownSm | js-sorting-indicator js-sorting-indicator-none %} <!-- PREVIEW ONLY -->
+                        … <!-- CODE ONLY -->
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr><td>Winter</td><td data-s-table-sort-val="12">December</td><td>2</td></tr>
+                <tr><td>Spring</td><td data-s-table-sort-val="3">March</td><td>13</td></tr>
+                <tr><td>Summer</td><td data-s-table-sort-val="6">June</td><td>25</td></tr>
+                <tr><td>Fall</td><td data-s-table-sort-val="9">September</td><td>13</td></tr>
+                <tr data-s-table-sort-to="bottom" class="fw-bold">
```

Comment:
Why isn't this row in a `<tfoot>`?  Is this just to demonstrate the functionality?

Reply:
Yep, that's all.